src/app/ui/figures/actions/action.ts

```diff
@@ -102,9 +102,12 @@ export class ActionComponent implements OnInit {
       return "+ " + this.action.value;
     } else if (this.action.valueType == ActionValueType.minus) {
       return "- " + this.action.value;
-    } else {
+    } else if (this.action.value !== undefined) {
       return this.action.value;
     }
+    else {
+      return ' ';
```

Comment:
When an empty string or undefined is returned the ui shows a '-' character. I couldn't work out why, I don't have a lot of frontend experience.